dhis2/src/main/resources/webapp/js/controllers.js

```diff
@@ -35,6 +35,13 @@
             };
 
             $scope.submit = function () {
+                if(!$scope.settings.serverURI.match("^https?:\/\/.+")) {
+                    if($scope.settings.serverURI.match("^.+:\/\/.+")) {
```

Comment:
I just wanted to replace other protocols with "http://". If it isn't need, I'll check only for http(s) occurence. If protocol will be absent I'll add it.